src/Uno.UI.Wasm/tsBindings/WindowManagerSetPointerEventsParams.ts

```diff
@@ -0,0 +1,20 @@
+/* TSBindingsGenerator Generated code -- this code is regenerated on each build */
+class WindowManagerSetPointerEventsParams
+{
+	/* Pack=4 */
+	HtmlId : number;
+	Enabled : boolean;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class property 'Enabled' must be marked either 'private', 'public', or 'protected'](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5247895)